packages/components/addon/components/hds/table/th-sort.hbs

```diff
@@ -1,6 +1,6 @@
-<th scope="col" class="hds-table__th-sort" aria-sort={{this.ariaSort}}>
+<th scope="col" class="hds-table__th-sort" aria-sort={{this.ariaSort}} data-test-table-th-sort>
```

Comment:
question related to this element: in the `Th` template I see that the `scope` is declared before the `...attributes` spreading. Is it OK if the consumers override this value? if not, then should be moved after it.

Reply:
Thought: we currently don't have any other component outputting `data-test-` attributes in the consumer's codebase. I see how it's being used in the test, but I guess we can get away with checking that the `hds-table__th-sort` class exists on a `th` element produced by this component – we do these sort of checks rather often in our codebase.

Reply:
@didoo this component doesn't support `...attributes` - @alex-ju that's why there's a data-test attribute.

Reply:
Also, I have focused more on the tests in the PR that follows this one: https://github.com/hashicorp/design-system/pull/650

Reply:
> @didoo this component doesn't support 

@MelSumner yes, I was referring to the `Th` (the non sortable one).